packages/ui-tests/cypress/support/next-commands/design.ts

```diff
@@ -65,11 +65,7 @@ Cypress.Commands.add('selectPrependNode', (nodeName: string, nodeIndex?: number)
 
 Cypress.Commands.add('performNodeAction', (nodeName: string, action: ActionType, nodeIndex?: number) => {
   nodeIndex = nodeIndex ?? 0;
-  cy.get(`[data-nodelabel="${nodeName}"]`)
-    .parent()
-    .eq(nodeIndex)
-    .find('g.pf-topology__node__action-icon > rect')
-    .click({ force: true });
+  cy.get(`[data-nodelabel="${nodeName}"]`).parent().eq(nodeIndex).rightclick({ force: true });
```

Comment:
@tplevko, I need to change this functionality from `getting the parent of the node to search for the three dots menu` to `right-click on the node` because of a structural change in `@patternfly/react-topology` that moves the selected node label to another layer, so we can't continue going to the parent and search for the menu there.